flat/settings.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+import django
+from django.conf import settings
+from django.core.exceptions import ImproperlyConfigured
+
+
+def check_installed_apps():
+    dj_version = django.VERSION
+    installed_apps = settings.INSTALLED_APPS
+
+    if dj_version < (1, 9):
```

Comment:
I don't see real use case for this. Obviously if someone installed 'flat' by pip and doesn't use it - we shouldn't warn user.

Let's use only `if dj_version > (1, 9):` case.